### PR TITLE
RAT Profile Localizable Strings

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
Text added for the antigen-test profile screen

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6618

### CODE REVIEW

**Review by naveeddotio — APPROVED** [review]

**Review by freshking — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -927,6 +927,20 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "ExposureSubmission_OverwriteNotice_Antigen_Text" = "Sie haben bereits einen Schnelltest registriert. Die App kann maximal einen Schnelltest und einen PCR-Test gleichzeitig verwalten. Wenn Sie einen weiteren Schnelltest registrieren, wird der erste Schnelltest aus der App gelöscht.";
 
+/* ExposureSubmission AntigenTest Profile Information */
+
+"ExposureSubmission_Antigen_Profile_Information_Title" = "Schnelltest-Profil";
+
+"ExposureSubmission_Antigen_Profile_Information_Image_Description" = "Eine QR Code der eine Person auf dem Smartphone identifiziert.";
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:934]: German should be "QR-Code" with a hyphen. 
"Eine QR-Code" sounds strange, I would say "Ein QR-Code".

```suggestion
"ExposureSubmission_Antigen_Profile_Information_Image_Description" = "Ein QR-Code der eine Person auf dem Smartphone identifiziert.";
```

@janetback can you confirm?

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -927,6 +927,20 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
+
+"ExposureSubmission_Antigen_Profile_Information_Description_Title" = "Legen Sie Ihre persönlichen Daten als QR-Code ab, um die Registrierung an der Teststelle  zu beschleunigen.";
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:936]: I think there is a double whitespace here before "zu beschleunigen" 
```suggestion
"ExposureSubmission_Antigen_Profile_Information_Description_Title" = "Legen Sie Ihre persönlichen Daten als QR-Code ab, um die Registrierung an der Teststelle zu beschleunigen.";
```

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (janetback) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:934]: Hi @KaiTeuber - I have just provided a screenreader text here for Android - could you please use the same one for iOS? Thanks
"ExposureSubmission_Antigen_Profile_Information_Image_Description" = "Eine Frau mit einem Smartphone in der Hand steht vor einem Gebäude. Ein QR-Code symbolisiert den zu scannenden Schnelltest-Profil.";
```

**Review by janetback — CHANGES_REQUESTED** [review]
Hi Kai, I have a change request for the screenreader text (see my comment). Thanks.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (janetback) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:934]: Hi @Ein-Tim, I have made an alternative text proposal :-) (see my comment)

Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:934]: Disregard this, @janetback made another proposal [here](https://github.com/corona-warn-app/cwa-app-ios/pull/2577#pullrequestreview-647938787)

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 2 above)
Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:936]: Could you still fix this @KaiTeuber? 

Thanks 🙂

**Review by janetback — APPROVED** [review]
UA reviewed

**Pull request merged**